src/xst.md

```diff
@@ -1,3 +1,90 @@
 # XST
 
-<!-- TODO: Add info -->
+- SORA Synthetics are tokens pegged to value indices, backed by XOR
+- The first SORA Synthetic Asset is the DAI-pegged XSTUSD
+- SORA Synthetic XST assets for other major stores of value will be available in the future, once oracles are integrated into the SORA network
+- XST was [proposed as an
+  RFP](https://github.com/sora-xor/rfps/pull/8/files) and is currently
+  on the SORA testnet
+
```

Comment:
not sure the list is a good choice here
I think a short paragraph summarizing the contents below would be a better tldr

Reply:
Updated!